app/src/main/res/layout/content_list_fragment.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools">
+  <data>
+    <variable
+      name="contentFragment"
+      type="org.oppia.app.player.content.ContentListFragment"/>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.content.ContentViewModel"/>
+  </data>
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:context=".player.content.ContentListFragment">
+    <androidx.recyclerview.widget.RecyclerView
+      android:id="@+id/recyclerview"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_alignParentTop="true"
```

Comment:
Given that this layout is in a constraint layout, is this needed?

Reply:
No its not required.